pkg/resources/test/fixtures/deployment-openstack-1.20.0-openstack-cloud-controller-manager-externalCloudProvider.yaml

```diff
@@ -65,7 +65,7 @@ spec:
         - '{"command":"/bin/openstack-cloud-controller-manager","args":["--kubeconfig=/etc/kubernetes/kubeconfig/kubeconfig","--v=1","--cloud-config=/etc/kubernetes/cloud/config","--cloud-provider=openstack"]}'
         command:
         - /http-prober-bin/http-prober
-        image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.21.1
+        image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.20.2
```

Comment:
NB: this is not a downgrade. It seems like we bumped this to v1.21 by mistake when adding support for Kubernetes 1.22.